tsdb/shard.go

```diff
@@ -447,6 +447,17 @@ func (s *Shard) validateSeriesAndFields(points []models.Point) ([]*FieldCreate,
 
 	// get the shard mutex for locally defined fields
 	for _, p := range points {
+		// verify the tags and fields
+		tags := p.Tags()
```

Comment:
We would also have that issue with the max series error. We may want to look into that again if we have the time.